src/workerd/io/compatibility-date.h

```diff
@@ -51,9 +51,33 @@ kj::Maybe<kj::String> normalizeCompatDate(kj::StringPtr date);
 // Returns the current date as a string formatted by CompatDate.
 kj::String currentDateStr();
 
-kj::Maybe<PythonSnapshotRelease::Reader> getPythonSnapshotRelease(
-    CompatibilityFlags::Reader featureFlags);
-kj::String getPythonBundleName(PythonSnapshotRelease::Reader pyodideRelease);
+class PythonRelease {
+ public:
+  PythonSnapshotRelease::Reader capnpRelease;
+  bool isDevelopment;
+  PythonRelease(PythonSnapshotRelease::Reader capnpRelease, bool isDevelopment)
+      : capnpRelease(capnpRelease),
+        isDevelopment(isDevelopment) {}
+
+  kj::StringPtr getPyodide() {
+    return capnpRelease.getPyodide();
+  }
+  kj::StringPtr getPyodideRevision() {
+    return capnpRelease.getPyodideRevision();
+  }
+  kj::StringPtr getPackages() {
+    return capnpRelease.getPackages();
+  }
+  kj::StringPtr getBaselineSnapshotHash() {
+    return capnpRelease.getBaselineSnapshotHash();
+  };
+  int64_t getBackport() {
+    return capnpRelease.getBackport();
+  }
+};
+
+kj::Maybe<PythonRelease> getPythonSnapshotRelease(CompatibilityFlags::Reader featureFlags);
```

Comment:
There's a key semantic difference between the old version of this function and the new.
The old had a reference-counting based copy constructor, the new does not.
That forced you to replace many instances of `PythonSnapshotRelease::Reader` with `PythonRelease&`.
In most places this is actually unsafe.
Seeing as PythonRelease is relatively small it might be appropriate to just implement a copy constructor for it and keep passing it by value as we did before and not by reference.

Reply:
I don't think anyone stores it ever?

Reply:
But it does make sense to implement copy and not worry about.